libvips/foreign/vips2webp.c

```diff
@@ -82,6 +83,73 @@ get_preset( VipsForeignWebpPreset preset )
 	return( -1 );
 }
 
+#if WEBP_ENCODER_ABI_VERSION < 0x0100
+typedef struct {
+	uint8_t *mem;
+	size_t size;
+	size_t max_size;
+	uint32_t pad[1];
```

Comment:
The whole code in this block is copied from libjpeg 0.1.99, it has only been changed to use the same formatting style as the rest of the file. This is done because 0.1.3 only has a private memory writer with a different data struct.